.yarnrc.yml

```diff
@@ -1,7 +1,7 @@
 nodeLinker: node-modules
 
 plugins:
-    - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
-      spec: '@yarnpkg/plugin-interactive-tools'
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
+    spec: "@yarnpkg/plugin-interactive-tools"
```

Comment:
If you run `prettier`, will change this file?

Reply:
![image](https://user-images.githubusercontent.com/579417/201914268-6d238816-3f51-4c43-aeb1-701e33f3befd.png)

hmm this should be formatted by the precommit hook

Reply:
looks like the hook isn't working (at least on my machine)

Thanks noticing it